posthog/models/event/query_event_list.py

```diff
@@ -210,6 +210,8 @@ def run_events_query(
 
     for expr in select:
         hogql_context.found_aggregation = False
+        if expr == "*":
+            expr = f'tuple({", ".join(SELECT_STAR_FROM_EVENTS_FIELDS)})'
```

Comment:
The events query expects a tuple when it gets a `"*"` field in its `select` array. This will all be refactored in https://github.com/PostHog/posthog/pull/14315